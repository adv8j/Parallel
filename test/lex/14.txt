@Task
{
    .ct(all)->5;.wt(t6,9);
}
