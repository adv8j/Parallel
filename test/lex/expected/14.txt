TASK
LBRACE
TASK_CHANNEL
LPAREN
ALL
RPAREN
ARROW
INT_LITERAL
SEMICOLON
CHANNEL_WAIT
LPAREN
IDENTIFIER
COMMA
INT_LITERAL
RPAREN
SEMICOLON
RBRACE
